### ISSUE TITLE
Jewelry indexes swapped by ZOS

### ISSUE THREAD

**Issue by Zeratoxx** [Rhyono/CraftStore#52]
The game internal indices for rings and necklaces got swapped by ZOS with the Gold Road chapter. 
Now, CraftStore shows rings in the necklace column and vice versa.

**Comment by Zeratoxx** [comment]
PR open: https://github.com/Rhyono/CraftStore/pull/51

**Comment by Zeratoxx** [comment]
Nested PR: https://github.com/Rhyono/CraftStore/pull/51 -> https://github.com/Sc4rlite/CraftStore/pull/1

**Comment by Zeratoxx** [comment]
@Rhyono i did a small mistake according the migration routine. I started the routine with the event `OnAddOnLoad` and wanted to create some chat messages as feedback. The routine gets executed successfully as far as I see, but no chat messages get written *IF* the game got freshly started! 
If you were already ingame and do a `/reloadui` then the chat messages get shown.. 

TLDR: The routine is presumably executed, but sometimes (depends) there is no feedback for the player.
